R/snapshot.R

```diff
@@ -791,6 +795,76 @@ renv_snapshot_description_source <- function(dcf) {
 
 }
 
+renv_snapshot_description_infer <- function(dcf) {
+
+  inferred <- tryCatch(
```

Comment:
Out of interest, why put this here instead of around `renv_remotes_resolve()` below?

Reply:
Mainly just to capture any kind of potential failure that could happen.

It also supports my crusade against excessive indentation, or having overly-large bodies in tryCatch blocks.